theme/templates/pages/page.html

```diff
@@ -16,8 +16,8 @@
 {% endmetablock %}{% endblock %}
 
 {% block title %}
-{#{% inplace_edit "page.title|linebreaksbr|safe" %}#}
-{{ page.title|linebreaksbr|safe }}
+{#{% inplace_edit "page.title|linebreaksbr" %}#}
+{{ page.title|linebreaksbr }}
```

Comment:
Good catch. I think I forgot about this one, but if I remember correctly it affects resource pages.